src/kafka.rs

```diff
@@ -7,6 +7,8 @@ use vector_config::configurable_component;
 
 use crate::{internal_events::KafkaStatisticsReceived, tls::TlsEnableableConfig};
 
+const PEM_START_MARKER: &str = "-----BEGIN ";
```

Comment:
This exact constant is already present in `lib/vector-core/src/tls/settings.rs`. I'd prefer you  make that a `pub` constant and reuse it.
https://github.com/vectordotdev/vector/blob/6b61a78eb3cb1d3c5c33c094339db645b1d9c767/lib/vector-core/src/tls/settings.rs#L26